src/index.ts

```diff
@@ -211,6 +211,9 @@ export type {
   Auth,
   DriverInfo,
   MongoOptions,
+  ServerApi,
```

Comment:
Ah just noticed, we don't have tooling to catch this unfortunately but api-extractor is satisfied when this is exported as a type, but if we want users to be able to use the enum it has to be exported normally, you can see on line 76 is where I grouped the "enums". 

Maybe a test/tooling we want to add (not here, just at some point) is importing index.js in our javascript tests and assert the actual imports we expect at runtime. 

(@durran just pinging you here for knowledge share since we all might run into this gap in tooling)